recipes/quilt/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "quilt3" %}
+{% set version = "3.0.5" %}
+
```

```suggestion
{% repo = "quilt" %}

```
`quilt3` is indeed the right package name, but it lives under `quilt` on GitHub.